generators/generator-base.js

```diff
@@ -437,6 +452,7 @@ module.exports = class extends Generator {
             { name: 'Dutch', value: 'nl' },
             { name: 'English', value: 'en' },
             { name: 'Estonian', value: 'et' },
+            { name: 'Farsi', value: 'fa' },
```

Comment:
add a new attribute `{ name: 'Farsi', value: 'fa', rtl: true },` and use that in `isRTLSupportNecessary` instead of hardcoding the values